activerecord/lib/active_record/associations/collection_proxy.rb

```diff
@@ -317,6 +317,13 @@ def build(attributes = {}, &block)
       end
       alias_method :new, :build
 
+      def insert_all(attributes, returning: nil, unique_by: nil)
```

Comment:
Just added for `insert_all` right now. We would need to solve it for `upsert_all`, `insert_all!`, `insert`, 'upsert' and `insert!` as well. If the approach I have done is correct, I'll add the rest of the methods

Reply:
I'd suggest using `**kwargs` here so that if the list of args (or their default values) change, we don't have to change them in two places.